src/index.ts

```diff
@@ -234,13 +238,17 @@ function parseAccountType(accountType: string): AccountType {
     return accountType as AccountType;
 }
 
-function parseAddress(address: string): string {
+function parseAddress(accountType: AccountType, address: string): string {
     if (_.isUndefined(address)) {
         throw new CLIError(CLIErrorType.OptionRequired, {
             optionName: "address"
         });
     }
-    // FIXME: Validate the address.
+    if (accountType === "platform") {
+        PlatformAddress.fromString(address);
+    } else {
+        AssetTransferAddress.fromString(address);
+    }
```

Comment:
We should catch error from `PlatformAddress.fromString` and `AssetTransferAddress.fromString`, and rethrow CLIError.
Also, we should add a proper enum value to `CLIErrorType`. And print a proper error message by adding the message in `getErrorMessage` in the error.ts file.

Reply:
@majecty Ok, I will fix it.